starport/pkg/nodetime/data/data_darwin_arm64.go

```diff
@@ -2,5 +2,5 @@ package data
 
 import _ "embed" // embed is required for binary embedding.
 
-//go:embed nodetime-darwin-amd64.tar.gz
```

Comment:
As mentioned in the PR itself, these will cause some builds to fail until we have merged in the PRs with arm binaries.